homework-submission/myapp/src/stores/rashadStore.js

```diff
@@ -0,0 +1,80 @@
+import { configure, action, observable, runInAction, computed } from "mobx";
+
+configure({ enforceActions: true });
+
+const url = "https://hyf-react-api.herokuapp.com";
+
+class RashadStore {
+  @observable todosList = [];
+
+  @action
+  checker = item => {
+    item.done = !item.done;
+    fetch(`${url}/todos/${item._id}`, {
+      method: "PATCH",
+      body: JSON.stringify({ done: item.done }),
```

Comment:
You can omit the line 12 and put it here directly `JSON.stringify({ done: !item.done })` just add the `!`